File: ce/snippets/visualbasic/CRMV8/entitiesvb/vb/createupdateentitymetadata9.vb
' <snippetcreateupdateentitymetadata9>



 Dim retrieveBankAccountEntityRequest As RetrieveEntityRequest = New RetrieveEntityRequest With {
  .EntityFilters = EntityFilters.Entity,
  .LogicalName = _customEntityName
 }
 Dim retrieveBankAccountEntityResponse As RetrieveEntityResponse = CType(_serviceProxy.Execute(retrieveBankAccountEntityRequest), RetrieveEntityResponse)
 Dim BankAccountEntity As EntityMetadata = retrieveBankAccountEntityResponse.EntityMetadata

 ' Disable Mail merge
 BankAccountEntity.IsMailMergeEnabled = New BooleanManagedProperty(False)
 ' Enable Notes
 Dim updateBankAccountRequest As UpdateEntityRequest = New UpdateEntityRequest With {
  .Entity = BankAccountEntity,
  .HasNotes = True
 }
 _serviceProxy.Execute(updateBankAccountRequest)

' </snippetcreateupdateentitymetadata9>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/entities/cs/createupdateentitymetadata9.cs
// <snippetcreateupdateentitymetadata9>



 RetrieveEntityRequest retrieveBankAccountEntityRequest = new RetrieveEntityRequest
 {
  EntityFilters = EntityFilters.Entity,
  LogicalName = _customEntityName
 };
 RetrieveEntityResponse retrieveBankAccountEntityResponse = (RetrieveEntityResponse)_serviceProxy.Execute(retrieveBankAccountEntityRequest);
 EntityMetadata BankAccountEntity = retrieveBankAccountEntityResponse.EntityMetadata;

 // Disable Mail merge
 BankAccountEntity.IsMailMergeEnabled = new BooleanManagedProperty(false);
 // Enable Notes
 UpdateEntityRequest updateBankAccountRequest = new UpdateEntityRequest
 {
  Entity = BankAccountEntity,
  HasNotes = true
 };



 _serviceProxy.Execute(updateBankAccountRequest);

// </snippetcreateupdateentitymetadata9>